cl_dll/view.cpp

```diff
@@ -78,7 +78,7 @@ extern cvar_t	*cl_forwardspeed;
 extern cvar_t	*chase_active;
 extern cvar_t	*scr_ofsx, *scr_ofsy, *scr_ofsz;
 extern cvar_t	*cl_vsmoothing;
-/*extern */ Vector   dead_viewangles;
+extern  Vector   dead_viewangles;
```

Comment:
I think we need a proper base where we can see changes. I realized that file very lately. Seeking through all of those non descripted commits isn't a good thing to do :)